_includes/pbc/all/install-features/202404.0/install-the-merchant-b2b-contracts-feature.md

```diff
@@ -0,0 +1,565 @@
+
+| PLUGIN                                                              | SPECIFICATION                                                                                          | PREREQUISITES                                                                                                                   | NAMESPACE                                                                  |
+|---------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| MerchantRelationshipHydratePlugin                                   | Expands that company user transfers that have a company business unit with the merchant relationship data. | Expects company users to have an assigned company business unit, which will be expanded with the merchant relationship data. | Spryker\Zed\MerchantRelationship\Communication\Plugin\CompanyUser          |
+| MerchantRelationshipDeleteMailTypeBuilderPlugin                     | Builds the `MailTransfer` with the data for the merchant relationship delete email.                             |                                                                                                                                 | Spryker\Zed\MerchantRelationship\Communication\Plugin\Mail                 |
+| CompanyBusinessUnitNotificationMerchantRelationshipPostDeletePlugin | Sends notification emails about deleted merchant relationships to company business units.             |                                                                                                                                 | Spryker\Zed\MerchantRelationship\Communication\Plugin\MerchantRelationship |
+
+**src/Pyz/Zed/CompanyUser/CompanyUserDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\CompanyUser;
+
+use Spryker\Zed\CompanyUser\CompanyUserDependencyProvider as SprykerCompanyUserDependencyProvider;
+use Spryker\Zed\MerchantRelationship\Communication\Plugin\CompanyUser\MerchantRelationshipHydratePlugin;
+
+class CompanyUserDependencyProvider extends SprykerCompanyUserDependencyProvider
+{
+    /**
+     * @return list<\Spryker\Zed\CompanyUserExtension\Dependency\Plugin\CompanyUserHydrationPluginInterface>
+     */
+    protected function getCompanyUserHydrationPlugins(): array
+    {
+        return [
+            new MerchantRelationshipHydratePlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+1. Create a merchant relationship.
+2. Log in as a company user who's a member of the business unit assigned to the relationship.
+3. Use the following code snippet to verify that the merchant relationship data is available in the company user transfer:
```

Comment:
What do I need to do with this snippet?

Reply:
Copy and paste it somewhere, it's just a code debug snippet.